package.json

```diff
@@ -22,18 +22,18 @@
     "*.js": "eslint --fix"
   },
   "dependencies": {
-    "moment": "^2.29.4",
-    "moment-timezone": "^0.5.35",
-    "probot": "^7.4.0",
-    "probot-scheduler": "^1.0.3"
+    "moment": "^2.30.1",
+    "moment-timezone": "^0.5.46",
+    "probot": "^13.4.1"
   },
   "devDependencies": {
+    "@octokit/rest": "^21.0.2",
```

Comment:
Is this dev dependency needed, as it doesn't seem we're using it anywhere?

Reply:
Good catch. Definitely not needed anymore.